src/components/Incidents/IncidentsPage.jsx

```diff
@@ -81,11 +76,10 @@ const IncidentsPage = ({
   React.useEffect(() => {
     setProcessedData(processIncidentsTimestamps(data));
   }, [data]);
```

Comment:
Rather than adding an extra use effect we can just set the processed data in the above useEffect